test/system/notifications_test.rb

```diff
@@ -50,7 +50,7 @@ class NotificationsTest < ApplicationSystemTestCase
                         path: '/reports/20400118',
                         user: users(:yamada),
                         sender: users(:machida))
-    visit_with_auth '/notifications/unread', 'yamada'
+    visit_with_auth '/notifications?query=unread', 'yamada'
```

Comment:
`query` -> `status` にしたので `status` っぽそう？

```suggestion
    visit_with_auth '/notifications?status=unread', 'yamada'
```

Reply:
このテストケースは「未読の通知ページに既読の通知が表示されないこと」をテストしているテストケースだと思うので、CI が通っちゃっているのはなにか変かもしれません。確認してもらえると良いかも。

Reply:
あーこれももしかしたら https://github.com/fjordllc/bootcamp/pull/3110/files#r698380992 の関係でテストが通っちゃっているのかも？

Reply:
https://github.com/fjordllc/bootcamp/pull/3110/files#r698380992 にも記載しましたが、提示していただいたコードで解決しました！